temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,67 @@
+package temporalcli
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"github.com/temporalio/cli/temporalcli/internal/trace"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalWorkflowTraceCommand) run(cctx *CommandContext, _ []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	ctx := cctx.Context
+
+	opts := trace.WorkflowTraceOptions{
+		Depth:       c.FlagDepth,
+		Concurrency: c.FlagConcurrency,
+		NoFold:      c.FlagNoFold,
+	}
+	opts.FoldStatus, err = trace.GetFoldStatus(c.FlagFold)
+	if err != nil {
+		return err
+	}
+
+	if err = c.printWorkflowSummary(cctx, cl, c.WorkflowId, c.RunId); err != nil {
+		return err
+	}
+	_, err = trace.PrintWorkflowTrace(ctx, cl, c.WorkflowId, c.RunId, opts)
+
+	return err
+}
+
+type workflowSummary struct {
```

Comment:
```suggestion
type workflowTraceSummary struct {
```

This package is shared by many commands, so `workflowSummary` specific to trace is too general of a name